app/src/marketplace/components/Modal/GetCryptoDialog/index.jsx

```diff
@@ -3,11 +3,12 @@
 import React from 'react'
 import { Translate, I18n } from 'react-redux-i18n'
 
-import WalletNoEthPng from '../../../assets/wallet_no_eth.png'
-import WalletNoEthPng2x from '../../../assets/wallet_no_eth@2x.png'
 import ModalPortal from '$shared/components/ModalPortal'
+import PngIcon from '$shared/components/PngIcon'
 import Dialog from '$shared/components/Dialog'
-import ExternalLinkButton from '$shared/components/Buttons/ExternalLinkButton'
+import Button from '$shared/components/Button'
+import Link from '$shared/components/Link'
+import { isMobile } from '$shared/utils/platform'
```

Comment:
Not used, can be removed.

<img width="816" alt="Screen Shot 2020-02-13 at 9 27 40" src="https://user-images.githubusercontent.com/1064982/74411306-96eb4d80-4e43-11ea-82f6-498fd98e450b.png">